pkgs/applications/graphics/write_stylus/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, qt5, makeWrapper, fetchurl, makeDesktopItem }:
```

Comment:
you should not add qt5 as an argument: see https://nixos.org/nixpkgs/manual/#ssec-qt-applications

Reply:
I just followed this guide https://nixos.wiki/wiki/Packaging/Binaries

If i remove qt5, the package don't build, tried to fix it with "https://nixos.wiki/wiki/Packaging/Binaries" but than i get undefined variable qt5 or undefined variable qtbase.

Reply:
this is a specificity of qt: if you use `libsForQt5.callPackage` instead of callPackage, you can add `qtbase` and other attributes of qt5 directly as an argument. See as an example https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/digitalbitbox/default.nix#L13-L16 and https://github.com/NixOS/nixpkgs/blob/release-18.03/pkgs/top-level/all-packages.nix#L124